### DISCUSSION TITLE
Blog comment on Building Multimodal AI in TypeScript

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#2421] [Docs/blog comments]
# Blog comment on Building Multimodal AI in TypeScript

We look at how to build Multimodal applications in TypeScript and dive into everything that needs to happen in between.

https://weaviate.io/blog/multimodal-search-in-typescript

**Comment by Salmon-Bard** [comment]
Can we add text as embedding to search for text also r use images to find text results?
